gamedata/cbasenpc.txt

```diff
@@ -209,13 +209,13 @@
 			"Studio_LookupSequence"
 			{
 				"library"		"server"
-				"windows"		"\x55\x8B\xEC\x56\x8B\x75\x08\x85\xF6\x75\x2A\x33\xC0\x5E\x5D\xC3\x8B\xCE\xE8\x2A\x2A\x2A\x2A\x84\xC0\x74\x2A\x53"
+				"windows"		"\x55\x8B\xEC\x83\xEC\x14\x80\x3D\x2A\x2A\x2A\x2A\x00\x0F\x85\x2A\x2A\x2A\x2A\x56\x57\x6A\x00\xC6\x05\x2A\x2A\x2A\x2A\x01\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xC7\x05\x2A\x2A\x2A\x2A\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\x83\xC4\x04\xA3\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\xFF\x35\x2A\x2A\x2A\x2A\xB9\x2A\x2A\x2A\x2A\xA3\x2A\x2A\x2A\x2A\xE8\x2A\x2A\x2A\x2A\x8B\xC8\xC1\xE1\x04\x2B\xC8\xA1\x2A\x2A\x2A\x2A\x8D\x34\x88\xC7\x46\x08\x2A\x2A\x2A\x2A\xC7\x06\x2A\x2A\x2A\x2A\xC7\x46\x04\x2A\x2A\x2A\x2A\xC7\x46\x0C\x05\x00\x00\x00"
```

Comment:
This doesn't seem correct. The function this points to looks to be an initialization function for some VScript bindings.

Reply:
Found it, use this one:
```suggestion
				"windows"		"\x55\x8B\xEC\x56\x8B\x75\x2A\x85\xF6\x0F\x84\x2A\x2A\x2A\x2A\x8B\xCE"
```

Reply:
Seems to be a bit of an oddball on Windows, maybe this one?
\x55\x8B\xEC\x56\x8B\x75\x08\x85\xF6\x0F\x84\x2A\x2A\x2A\x2A\x8B\xCE

Reply:
Oops, forgot to click the button.